<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AgileStock Interface</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #1E447B;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="title_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="title_spacer_left">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="agilestock_title_label">
         <property name="styleSheet">
          <string notr="true">color:#ffffff;
font-family: &quot;Orbitron&quot;, sans-serif;
font-size: 28px;</string>
         </property>
         <property name="text">
          <string>AgileStock</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="title_spacer_right">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="central_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QFrame" name="upper_frame">
         <property name="styleSheet">
          <string notr="true">background-color: #293C58;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,0,3">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QFrame" name="barcode_scanner_frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(250, 251,252, .85);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <widget class="QFrame" name="barcode_comm_status_frame">
               <property name="styleSheet">
                <string notr="true">background-color: #FAFBFC;
border-bottom-right-radius: 4px;
border-top-left-radius: 4px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,2">
                <property name="leftMargin">
                 <number>4</number>
                </property>
                <property name="topMargin">
                 <number>4</number>
                </property>
                <property name="rightMargin">
                 <number>4</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="barcode_scanner_icon">
                  <property name="styleSheet">
                   <string notr="true">background-color: none;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>barcode-scanner-64.png</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="barcode_scanner_comm_label">
                  <property name="styleSheet">
                   <string notr="true">color: White;
font-family: &quot;SourceSansPro-Bold&quot;;
font-weight: 600;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DBE9F3, stop: 0.1 #FF0000, stop: 0.5 #FF0000, stop: 1 #E90B0B, stop: 1 #BB0009);
font-size: 14px;
border-radius: 4px;</string>
                  </property>
                  <property name="text">
                   <string>SR-1000</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QFrame" name="barcode_information_frame">
               <property name="styleSheet">
                <string notr="true">background-color: #67717E;
border-top-right-radius: 4px;
border-bottom-left-radius: 4px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>4</number>
                </property>
                <property name="topMargin">
                 <number>4</number>
                </property>
                <property name="rightMargin">
                 <number>4</number>
                </property>
                <property name="bottomMargin">
                 <number>4</number>
                </property>
                <item>
                 <widget class="QLabel" name="barcode_status_static_label">
                  <property name="styleSheet">
                   <string notr="true">color: #F9F9F9;
font-family: &quot;SourceSansPro-Bold&quot;;
font-weight: 600;
background-color: none;
font-size: 14px;</string>
                  </property>
                  <property name="text">
                   <string>Barcode Status Message</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="barcode_status_dynamic_label">
                  <property name="styleSheet">
                   <string notr="true">color: Black;
font-family: &quot;SourceSansPro-Bold&quot;;
font-weight: 600;
background-color: none;
font-size: 12px;
background-color: #fdfdfd;
border-radius: 4px;</string>
                  </property>
                  <property name="text">
                   <string>Read Unsuccessful</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QFrame" name="read_barcode_frame">
               <property name="styleSheet">
                <string notr="true">background-color: #95BDE4;
border-top-left-radius: 4px;
border-bottom-right-radius: 4px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>4</number>
                </property>
                <property name="topMargin">
                 <number>4</number>
                </property>
                <property name="rightMargin">
                 <number>4</number>
                </property>
                <property name="bottomMargin">
                 <number>4</number>
                </property>
                <item>
                 <widget class="QLabel" name="item_barcode_static_label">
                  <property name="styleSheet">
                   <string notr="true">color: Black;
font-family: &quot;SourceSansPro-Bold&quot;;
font-weight: 600;
background-color: none;
font-size: 14px;</string>
                  </property>
                  <property name="text">
                   <string>Item Barcode</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="item_barcode_dynamic_label">
                  <property name="styleSheet">
                   <string notr="true">color: Black;
font-family: &quot;SourceSansPro-Bold&quot;;
font-weight: 600;
background-color: none;
font-size: 12px;
background-color: #ffffff;
border-radius: 4px;</string>
                  </property>
                  <property name="text">
                   <string>{NULL}</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="scanner_command_frame">
               <property name="styleSheet">
                <string notr="true">background-color: #5E768F;
border-top-right-radius: 4px;
border-bottom-left-radius: 4px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>4</number>
                </property>
                <property name="topMargin">
                 <number>4</number>
                </property>
                <property name="rightMargin">
                 <number>4</number>
                </property>
                <property name="bottomMargin">
                 <number>4</number>
                </property>
                <item>
                 <widget class="QPushButton" name="start_scanning_button">
                  <property name="styleSheet">
                   <string notr="true">QPushButton#start_scanning_button
{	

color: White;
	font-family: &quot;SourceSansPro-Bold&quot;;
	font-weight: 600;
	font-size: 10px;


	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #98FF99, stop: 0.1 #78C26B, stop: 0.5 #4DB044, stop: 1 #4DB044, stop: 1 #248423);
	border-style: outset;
	border-width: 1px;
	border-color:black;
	padding: 6px;
	outline-offset: 4px;
	border-radius: 4px;
}

QPushButton:hover:!pressed#start_scanning_button
{
	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #335A44, stop: 0.1 #78C26B, stop: 0.5 #4DB044, stop: 1 #4DB044, stop: 1 #335A44);
	border-color:white;
}
</string>
                  </property>
                  <property name="text">
                   <string>Start Scanning</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stop_scanning_button">
                  <property name="styleSheet">
                   <string notr="true">QPushButton#stop_scanning_button
{	

	color: White;
	font-family: &quot;SourceSansPro-Bold&quot;;
	font-weight: 600;
	font-size: 10px;


	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFD1AD, stop: 0.1 #FF8500, stop: 0.5 #FF7500 stop: 1 #FF6500, stop: 1 #FF5500);
	border-style: outset;
	border-width: 1px;
	border-color:black;
	padding: 6px;
	outline-offset: 4px;
	border-radius: 4px;
}

QPushButton:hover:!pressed#stop_scanning_button
{
	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #C46A33, stop: 0.1 #FF8500, stop: 0.5 #FF7500 stop: 1 #FF6500, stop: 1 #C46A33);
	border-color:white;
}
</string>
                  </property>
                  <property name="text">
                   <string>Stop Scanning</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="navigation_frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(94,118,143, .85);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QPushButton" name="add_items_button">
               <property name="styleSheet">
                <string notr="true">QPushButton#add_items_button
{	

color: White;
	font-family: &quot;SourceSansPro-Bold&quot;;
	font-weight: 600;
	font-size: 10px;


	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #98FF99, stop: 0.1 #78C26B, stop: 0.5 #4DB044, stop: 1 #4DB044, stop: 1 #248423);
	border-style: outset;
	border-width: 1px;
	border-color:black;
	padding: 6px;
	outline-offset: 4px;
	border-radius: 4px;
}

QPushButton:hover:!pressed#add_items_button
{
	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #335A44, stop: 0.1 #78C26B, stop: 0.5 #4DB044, stop: 1 #4DB044, stop: 1 #335A44);
	border-color:white;
}
</string>
               </property>
               <property name="text">
                <string>Add Items</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_item_button">
               <property name="styleSheet">
                <string notr="true">QPushButton#remove_item_button
{	

color: White;
	font-family: &quot;SourceSansPro-Bold&quot;;
	font-weight: 600;
	font-size: 10px;


	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DBE9F3, stop: 0.1 #FF0000, stop: 0.5 #FF0000, stop: 1 #E90B0B, stop: 1 #BB0009);
	border-style: outset;
	border-width: 1px;
	border-color:black;
	padding: 6px;
	outline-offset: 4px;
	border-radius: 4px;
}

QPushButton:hover:!pressed#remove_item_button
{
	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #9A1505, stop: 0.1 #FF0000, stop: 0.5 #FF0000, stop: 1 #C71E08, stop: 1 #62130A);
	border-color:white;
}
</string>
               </property>
               <property name="text">
                <string>Remove Items</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="launch_website_button">
               <property name="styleSheet">
                <string notr="true">QPushButton#launch_website_button
{	

color: black;
	font-family: &quot;SourceSansPro-Bold&quot;;
	font-weight: 600;
	font-size: 12px;


	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #d6d7d8, stop: 0.1 #c6c6c8, stop: 0.5 #c6c6c8, stop: 1 #b5b6b9, stop: 1 #8a8b8e);
	border-style: outset;
	border-width: 1px;
	border-color:black;
	padding: 6px;
	outline-offset: 4px;
	border-radius: 4px;
}

QPushButton:hover:!pressed#launch_website_button
{
	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #d6d7d8, stop: 0.1 #c6c6c8, stop: 0.5 #c6c6c8, stop: 1 #b5b6b9, stop: 1 #575759);
	border-color: orange;
}
</string>
               </property>
               <property name="text">
                <string>Launch Website</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="database_actions_stacked_widget">
            <widget class="QWidget" name="add_item_pane"/>
            <widget class="QWidget" name="remove_item_pane">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="lower_frame">
         <property name="styleSheet">
          <string notr="true">background-color: #293C58;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
